api/envoy/config/health_checker/mysql/v2/BUILD

```diff
@@ -0,0 +1,8 @@
+load("//bazel:api_build_system.bzl", "api_proto_library")
```

Comment:
Check out the recent thrift PR. Can we add this extension to CODEOWNERS?

Reply:
that might have been added by mistake while rebasing

Reply:
nvm, I've mixed things up in my head

Reply:
https://github.com/envoyproxy/envoy/blob/master/CODEOWNERS. See the new extension policy. Each extension needs owners that will handle code reviews.

Reply:
I can add it to that file. Who should I put up as owner?

Reply:
:)
kinda saw that coming, but as I transition to a new team I won't have the cycles to spare, unfortunately

Reply:
@juchem is there someone on your old team who would want to help maintain this? If not, another option is to just keep it as a private or public out of tree filter for now.

Reply:
@austin-zhu @Jason-Jian @ken-experiment any interest in helping maintain the MySQL health check?

Reply:
Sign me up. although I'd need to come up to speed first. :)

Reply:
There you go. I'll spend some time this week with Jason to provide an overview of the code structure, then I'll add him as a maintainer of the MySQL health checker in the `CODEOWNERS` file.